tensorboard/compat/proto/update.sh

```diff
@@ -26,10 +26,10 @@ rsync --existing "$1"/tensorflow/core/protobuf/*.proto tensorboard/compat/proto/
 rsync --existing "$1"/tensorflow/core/profiler/*.proto tensorboard/compat/proto/
 rsync --existing "$1"/tensorflow/core/util/*.proto tensorboard/compat/proto/
 rsync --existing "$1"/tensorflow/python/framework/*.proto tensorboard/compat/proto/
-if [ -d $1"/tensorflow/tsl/protobuf" ] # This directory first appears in TF 2.11
-then
-  rsync --existing "$1"/tensorflow/tsl/protobuf/*.proto tensorboard/compat/proto/
-fi
+# The "TSL" protos are now in their  own repo, but they are included as a
+# "vendored" package within TF. Therefore, this dir corresponds to the
+# tsl/protobuf path that is replaced below, without the tensorflow/ prefix.
```

Comment:
For what it's worth I think the part of the sentence ", without the tensorflow/ prefix" is more confusing than helpful. It could just be removed.

I read it as "somewhere tsl/protobuf has a tensorflow prefix but here it does not", which is not true. I think you meant it in a different sense (maybe to help the user find L43?) but it doesn't clearly read that way.

Reply:
Updated.